commit 37b46e91bdb51e6116df9a7edc2eb79206237f1c
Author: Panu Matilainen <pmatilai@redhat.com>
Date:   Fri Dec 11 14:32:38 2015 +0200

    scripts: fix relpath.sh output when $prefix is set in environment
    
    When relpath.sh is called from install target with prefix set, eg
    "make install DESTDIR=/tmp/dpdk-root prefix=/usr", the prefix
    from the environment leaks to relpath.sh internal helper variable and
    causes incorrect symlinks to be generated in sdk $(targetdir):
    
       include -> /usr../../../include/dpdk
       lib -> /usr../../../lib
    
    Initialize the local variable to empty to avoid side-effects from
    environment.
    
    Signed-off-by: Panu Matilainen <pmatilai@redhat.com>

diff --git a/scripts/relpath.sh b/scripts/relpath.sh
index 7d2f48f..4ff4671 100755
--- a/scripts/relpath.sh
+++ b/scripts/relpath.sh
@@ -61,6 +61,8 @@ right2=${REL2#*/}
 prev_right2=$REL2
 prev_left2=
 
+prefix=
+
 while [ "${right1}" != "" -a "${right2}" != "" ]; do
 
 	if [ "$left1" != "$left2" ]; then
